test/functional/mempool_sigoplimit.py

```diff
@@ -165,7 +165,8 @@ def create_bare_multisig_tx(utxo_to_spend=None):
         # But together, it's exceeding limits in the *package* context. If sigops adjusted vsize wasn't being checked
         # here, it would get further in validation and give too-long-mempool-chain error instead.
         packet_test = self.nodes[0].testmempoolaccept([tx_parent.serialize().hex(), tx_child.serialize().hex()])
-        assert_equal([x["package-error"] for x in packet_test], ["package-mempool-limits", "package-mempool-limits"])
+        expected_package_error = f"package-mempool-limits, package size {2*20*5000} exceeds ancestor size limit [limit: 101000]"
```

Comment:
The magic number can be declared with a descriptive variable name,
MAX_PUBKEYS_PER_MULTISIG = 20
bytespersigop = 5000

We can reuse them in the same function
```suggestion
expected_package_error = f"package-mempool-limits, package size {2*MAX_PUBKEYS_PER_MULTISIG*bytespersigop} exceeds ancestor size limit [limit: 101000]"
```

Reply:
Done in #29424